src/platforms/java/guides/spring-boot/logging-frameworks.mdx

```diff
@@ -67,7 +67,16 @@ However, if errors that may appear during startup should to be sent to Sentry, t
 
 ### Log4j2
 
-To use Sentry's Log4j2 integration in Spring Boot application, [follow the guide on configuring Log4j2 with Spring Boot](https://docs.spring.io/spring-boot/docs/current/reference/html/howto.html#howto-configure-log4j-for-logging) and configure `SentryAppender` in the `log4j2.xml` file:
+To use Sentry's Log4j2 integration in Spring Boot application, you must include a dependency to the `sentry-log4j` module:
+
+```xml {tabTitle:Maven}
```

Comment:
ups the suggestion messed up because of the backticks, the suggestion is:

> ```xml {tabTitle:Maven}
> <dependency>
>     <groupId>io.sentry</groupId>
>     <artifactId>sentry-log4j2</artifactId>
>     <version>{{ packages.version('sentry.java.log4j2', '4.2.0') }}</version>
> </dependency>
> ```
> 
> ```groovy {tabTitle:Gradle}
> implementation 'io.sentry:sentry-log4j2:{{ packages.version('sentry.java.log4j2', '4.2.0') }}'
> ```
> 
> ```scala {tabTitle: SBT}
> libraryDependencies += "io.sentry" % "sentry-log4j2" % "{{ packages.version('sentry.java.log4j2', '4.2.0') }}"
> ```

Reply:
similar to https://github.com/getsentry/sentry-docs/blob/master/src/includes/getting-started-install/java.log4j2.mdx 
can you fix @kevinparaport ?
thanks for doing this